src/main.py

```diff
@@ -1 +1,87 @@
-# Your code lives here, Good luck!
\ No newline at end of file
+### packages importing
+import os
+import cv2
+import numpy as np
+import json
+import jiwer
+import pytesseract
+import time
+
+
+### input images reading
+def read_img_from_source():
+  images_array = []
+  absolute_input_path = os.path.abspath("../input")
+  input_images_nouns = os.listdir(absolute_input_path)
+  input_images_nouns.remove(".ipynb_checkpoints")

```

Comment:
Crashes if the file is not in the directory. To make sure you only read images you could have forced a check before:
```python
input_images_nouns = [input_images_noun for input_images_noun in input_images_nouns if input_images_noun.endswith((".png", ".jpg", ".jpeg"))]
```